minio/minioadmin.py

```
@@ -763,7 +782,7 @@ def add_service_account(self,
             raise ValueError("both access key and secret key must be provided")
```

Add xor check of `policy_file` and `policy` here and fail early.